TKGame/Item.cs

```diff
@@ -54,9 +54,30 @@ private Item()
         /// <param name="gameTime"></param>
         public override void Update(GameTime gameTime)
         {
-            Player player = EntityManager.GetEntities().FirstOrDefault(x => x.entityName == "Player" && x is Player) as Player;
-            // TODO: check if player is null
+            PickUpItem();
+        }
+
+        /// <summary>
+        /// Method to simulate picking up item
+        /// Checks if player is within range of item,
+        /// if so, then transports item to corner
+        /// to simulate inventory
+        /// </summary>
+        public void PickUpItem()
+        {
+            Player player = Player.Instance;
 
+            if (player != null)
+            {
+                // if player is within range of item spawn position
+                if (player.Position.X >= Position.X - 20 &&
```

Comment:
May want to set some member variables in the class to remove the magic numbers later on.